frontend/apps/reader/modules/readerrolling.lua

```diff
@@ -1180,14 +1180,14 @@ function ReaderRolling:showEngineProgress(percent)
     end
 
     if percent then
-        local now = TimeVal:now()
-        if self.engine_progress_update_not_before and now < self.engine_progress_update_not_before then
+        local now_fts = fts.now()
+        if self.engine_progress_update_not_before_fts and now_fts < self.engine_progress_update_not_before_fts then
```

Comment:
Random pick (not really looking at the whole thing):
"fts" reads less obvious than "timeval" or "time".
I understand it's helpful for now for you to catch what's left to be migrated.
But, after/if this gets merged, seeing `_fts` everywhere will make things ugly - and I'd rather keep reading:
```lua
local now_fts = time.now() -- or whatever you end up naming it, Time, TimeVal, but something more obvious than 'fts'
if self.engine_progress_update_not_before and now < self.engine_progress_update_not_before then
```
as in here when all is migrated, we don't care about the internal representation, TimeVal or "fixed time second".

No real opinion about the existing internal reprensentation and the suggested change.
But interested in anything else to read as english and simple arithmetic to keep working :)

Reply:
Just some thoughts about naming convention (maybe for the docs): 
I would like to rename fts to time! (as a reminiscense to TimeVal)

If the name implies a time (now, when, until, xxxdeadline, xxxtime, getElapsedTimeSinceBoot, lastxxxtimexxx, ...) we assume this value to be a time (fts encoded).

Other objects which are times (like last_tap,  
 tap_interval_override, ...) can be renamed to last_tap_time (and hence be obvious  fts-encoded too) easily.

All other time objects (a handfull) get the appropriate suffix _ms, _us, _s (_m, _h, _d ) to show that they are special.

So objects would have the same names most of the times (haha :) ), sometimes get a clearification "_time"  👍 (or a renaming with xxxtimexxx) and very very seldom get a suffix (to improve the correct  usage 💯 ).